webapp/src/app.tsx

```diff
@@ -1991,14 +1991,19 @@ function setHidPollerDelay() {
 }
 
 function initSerial() {
+    startHidConnectionPoller();
```

Comment:
This causes an infinite loop here where initSerial is calling startHidConnectionpoller, and startHidConnectionpoller is calling initSerial

Reply:
That seems like the intended behavior: every 5 seconds, scan devices which may have dissapeared or popped up.

We should move this stuff out of app.tsx and clean up to avoid confusion.

Reply:
Yeah intended behaviour is to scan every 5 seconds, but here's what happens: 
- initSerial calls startHid which sets an interval to call initSerial every 5 seconds. 
- every time that runs, it calls initSerial and then that sets a new interval which then runs every 5 seconds. 

You get the idea. Basically should only set the interval if hasn't already been set..